controller/telemetry/server.go

```diff
@@ -46,6 +47,15 @@ var (
 		responseLabels,
 	)
 
+	latencyBucketMaxValues = [26]uint32{
```

Comment:
this array is (almost) identical to `responseLatencyBuckets` below.  We shouldn't have both.